### PR TITLE
UI: add preferred node version, ignore prettier formatting

### PR DESCRIPTION
**Pull request by jayeshchoudhary**
This is `ui`, `chore`

### CODE REVIEW

**Comment by jayeshchoudhary** [comment]
@saurabhd336 @jadami10  please take a look

**Comment by jayeshchoudhary** [comment]
@saurabhd336 @jadami10 please take a look

File: pinot-controller/src/main/resources/.prettierignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*
```
Comment (jadami10) [inline pinot-controller/src/main/resources/.prettierignore:1]: what's the reasoning here? If anything I was hoping we'd start enforcing prettier

**Review by jadami10 — APPROVED** [review]
just one comment, but fine either way since we don't enforce prettier anywhere

File: pinot-controller/src/main/resources/.prettierignore (hunk 1 above)
Comment (jayeshchoudhary) [reply pinot-controller/src/main/resources/.prettierignore:1]: I will write prettier and eslint configs properly for this project.
That will be in a separate PR, until then this should be ok.

**Review by jadami10 — APPROVED** [review]

**Comment by Jackie-Jiang** [comment]
Can you please take a look at the failed tests

**Comment by jayeshchoudhary** [comment]
addressed here 
https://github.com/apache/pinot/pull/12924

**Pull request closed without merging**